### PR TITLE
All plots sections visible

### PR DESCRIPTION
**Pull request by sroy3**
Part of #1933 
Screen Shot 2022-08-04 at 3 06 09 PM
Screen Shot 2022-08-04 at 3 05 30 PM
Screen Shot 2022-08-04 at 3 04 32 PM

### CODE REVIEW

File: webview/package.json [hunk 1]
```diff
@@ -47,7 +47,6 @@
     "@storybook/manager-webpack5": "6.5.9",
     "@storybook/preset-scss": "1.0.3",
     "@storybook/react": "6.5.9",
-    "@storybook/testing-library": "0.0.13",
```
Comment (sroy3) [inline webview/package.json:50]: This is there twice in the list

Comment (mattseddon) [inline webview/package.json:50]: I think you'll need to pin the other version (maybe just remove that one as it is out of order)

File: webview/src/plots/components/App.test.tsx [hunk 2]
```diff
@@ -192,29 +212,27 @@ describe('App', () => {
...
-    })
-
-    sendSetDataMessage({
       template: templatePlotsFixture
     })
 
     expect(screen.queryByText('Loading Plots...')).not.toBeInTheDocument()
     expect(screen.getByText('Trends')).toBeInTheDocument()
-    expect(screen.getByText('Data Series')).toBeInTheDocument()
+    expect(screen.getByText('No Metrics Selected')).toBeInTheDocument()
```
Comment (mattseddon) [inline webview/src/plots/components/App.test.tsx:235]: [Q] Can we hide the context menu dropdown(s) when there are no actual plots for the project that we are showing? I think that at the moment the "filter metrics" dropdown would show as empty?

File: webview/src/plots/components/App.test.tsx [hunk 3]
```diff
@@ -246,6 +264,7 @@ describe('App', () => {
   })
 
   it('should toggle the checkpoint plots section in state when its header is clicked', async () => {
+    // Fails when other test above
```
Comment (mattseddon) [inline webview/src/plots/components/App.test.tsx:267]: [Q] Missed this one?

File: webview/src/plots/components/App.test.tsx [hunk 4]
```diff
@@ -1108,7 +1147,11 @@ describe('App', () => {
 
         beforeEach(async () => {
           // eslint-disable-next-line testing-library/no-render-in-setup
```
Comment (mattseddon) [inline webview/src/plots/components/App.test.tsx:1149]: [C] If we're ok with this we should disable the rule globally.

**Review by mattseddon — APPROVED** [review]
For checkpoint plots, I think we need to differentiate between no metrics being selected and there being no plots. Extending that we should probably have different text based on whether or not the project has those types of plots.

**Comment by mattseddon** [comment]
@sroy3 could be unrelated to this particular change but virtualized plots seems broken in the storybook

**Review by julieg18 — APPROVED** [review]

**Comment by sroy3** [comment]
> @sroy3 could be unrelated to this particular change but virtualized plots seems broken in the storybook

The only difference is the Images section being added between the two other sections.

**Comment by sroy3** [comment]
> For checkpoint plots, I think we need to differentiate between no metrics being selected and there being no plots. Extending that we should probably have different text based on whether or not the project has those types of plots.

Changed it for when there is no data for checkpoint plots

File: webview/src/plots/components/App.test.tsx (hunk 4 above)
Comment (sroy3) [reply webview/src/plots/components/App.test.tsx]: Done

File: webview/src/plots/components/App.test.tsx (hunk 3 above)
Comment (sroy3) [reply webview/src/plots/components/App.test.tsx]: 🤦🏼‍♀️

File: webview/src/plots/components/App.test.tsx (hunk 2 above)
Comment (sroy3) [reply webview/src/plots/components/App.test.tsx]: Done

**Pull request merged**